resources/assets/components/utilities/ScholarshipCard/ScholarshipCard.js

```diff
@@ -49,6 +55,19 @@ const ScholarshipCard = ({ campaign }) => {
     { height: 820, width: 1460 },
   ]);
 
+  const handleScholarshipCardShareClick = () => {
+    trackAnalyticsEvent('clicked_scholarship_gallery_block_apply_now', {
+      action: 'button_clicked',
+      category: EVENT_CATEGORIES.siteAction,
+      label: 'scholarship_gallery_card',
+      context: {
+        url: path,
+        campaignId,
+        ...getPageContext(),
```

Comment:
Awesome! And last thing, this just needs to be flipped around so that the `campaignId` we're providing here manually, will overwrite the one returned from `...getPageContext()`:
```suggestion
        ...getPageContext(),
        campaignId,
```